relayer/relayer/src/connections/azero.rs

```diff
@@ -1,3 +1,5 @@
+use std::sync::{Arc, Mutex as SyncMutex};
```

Comment:
Is there some good reason here to use Mutex from std instead of tokio Mutex?

Reply:
2nd case I think we should only increase the nonce if we were successful in submitin it to the chain. I can use the tokio mutex, np

Reply:
I think the sync version has less overhead, even in async scenarios (read some articles on that).

Reply:
after discusion changed it to Atomicu64